File: 5523.c
#include <stdio.h>

int main(){
  int A_win = 0;
  int B_win = 0;
  int A_s, B_s, Test_n;
  scanf("%d", &Test_n);
  while (Test_n-- > 0){
    scanf("%d %d", &A_s, &B_s );
    if (A_s > B_s){
      A_win += 1;
    }
    else if (A_s < B_s){
      B_win += 1;
    }
  }
  printf("%d %d\n", A_win, B_win);
}

File: Python/5523.py
count = 0
B_win = 0
A_win = 0
round = int(input())
while count < round :
    S = input().partition(" ")
    A = int(S[0])
    B = int(S[2])
    if A > B :
        A_win += 1
        count += 1
    elif B > A :
        B_win += 1
        count += 1
    else :
        count += 1
print(A_win, B_win)
